dev/com.ibm.ws.webcontainer.security/resources/com/ibm/ws/webcontainer/security/resources/WebAppSecurityMessages.nlsprops

```diff
@@ -149,3 +149,7 @@ JWT_ALREADY_LOGGED_OUT.useraction=Obtain a new JSON Web Token.
 JWT_COOKIE_SECURITY_MISMATCH=CWWKS9127W: The jwtSso feature is setting a secure cookie on an insecure HTTP request.  This might be a configuration error.  Use the HTTPS protocol or set the jwtSso configuration attribute setCookieSecureFlag to false.
 JWT_COOKIE_SECURITY_MISMATCH.explanation=A cookie is marked as secure but the protocol is HTTP, so the cookie will never be returned, this might cause subsequent requests to fail.
 JWT_COOKIE_SECURITY_MISMATCH.useraction=Use the HTTPS protocol or set the jwtSso configuration attribute setCookieSecureFlag to false.
+
+TOKEN_EXPIRATION_MISMATCH=CWWKS9128W: JWT and LTPA cookies are both being set on a response, with differing expiration times in minutes of {0} and {1}. This might be a configuration error. Change the Ltpa configuration or JwtSso jwtBuilderRef configuration so both types of tokens have the same expiration time.
+TOKEN_EXPIRATION_MISMATCH.explanation=Tokens with differing expiration times might cause subsequent requests to fail unexpectedly. 
+TOKEN_EXPIRATION_MISMATCH.useraction=Change the Ltpa configuration or JwtSso jwtBuilderRef configuration so both types of tokens have the same expiration time.  
```

Comment:
Suggested edits below:
TOKEN_EXPIRATION_MISMATCH=CWWKS9128W: JWT and LTPA cookies are both set on a response, with differing expiration times of {0} and {1} minutes. Change the ltpa or jwtBuilder configuration elements so that both types of tokens have the same expiration time.
TOKEN_EXPIRATION_MISMATCH.explanation=JSON Web Token (JWT) and Lightweight Third Party Authentication (LTPA) tokens are both enabled in the server configuration, but they have different expiration times. Tokens with differing expiration times might cause subsequent requests to fail unexpectedly. 
TOKEN_EXPIRATION_MISMATCH.useraction=Configure JWT and LTPA tokens to have the same expiration time. For LTPA, change the configuration of the ltpa element. For JWT, change the configuration of the jwtBuilderRef element that is referenced in your jwtSso configuration.  

**Question:** I noticed in the documentation that LTPA expiration seems to be set in hours rather than minutes: https://www.ibm.com/support/knowledgecenter/was_beta_liberty/com.ibm.websphere.liberty.autogen.nd.doc/ae/rwlp_config_jwtBuilder.html Will the values in the message be displayed in minutes anyway?

Reply:
Due to technical difficulties I'm going to move the message and the code that emits it to a later pull, so you can approve or unassign, as appropriate.

Reply:
actually I'll make your updates so we don't lose them, but comment them out for now.